### DISCUSSION TITLE
Osu! mouse pointer goes out of screen

### DISCUSSION THREAD

**Discussion by danielshamsudin** [ppy/osu#27673] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Currently running GNOME on wayland and the mouse seems like not mapping properly to the whole screen / window. I'm using dual monitor config if it helps. 

OS: EndeavorOS
Kernel: 6.7.9-273-tkg-pds
CPU: AMD R7 3700U

### Version

20240123

### Logs

[session.log](https://github.com/ppy/osu/files/14654154/session.log)
[update_success.log](https://github.com/ppy/osu/files/14654155/update_success.log)
[runtime.log](https://github.com/ppy/osu/files/14654156/runtime.log)
[osu!auth.log](https://github.com/ppy/osu/files/14654157/osu.auth.log)

**Comment by bdach** [comment]
If you're running wayland this could be a whole mess of things that we can't easily debug without having access to your specific software configuration.

**Reply by danielshamsudin** [reply]
may I know what can I provide in order to troubleshoot further?

**Reply by bdach** [reply]
Regretfully we don't have the dev time to debug people's linux setups. Your best bet is debugging yourself.

**Reply by danielshamsudin** [reply, marked as answer]
Just to drop an update, using osu! lazer works OOTB from AUR. Thanks for the help regardless!
